src/python/dxpy/templating/templates/nextflow/src/nextflow.sh

```diff
@@ -140,7 +103,6 @@ main() {
     run @@RESOURCES_SUBPATH@@ \
     $profile_arg \
     -name ${DX_JOB_ID}${env_job_suffix} \
-    $RESUME_CMD \
```

Comment:
Not used here, and not defined here yet (this is the initial mock-run for processing configuration).

Reply:
Opening a separate task for that --> APPS-2699